libraries/ESP8266WiFi/examples/BearSSL_CertStore/certs-from-mozilla.py

```diff
@@ -20,8 +22,11 @@
     from StringIO import StringIO
 except Exception:
     from io import StringIO
-
-# Mozilla's URL for the CSV file with included PEM certs
+#check if ar is available
+if which('ar') is None and not os.path.isfile('./ar') and not os.path.isfile('./ar.exe'):
+    raise Exception("You need the program 'ar' from xtensa-lx106-elf found here: (esp8266-arduino-core)/hardware/esp8266com/esp8266/tools/xtensa-lx106-elf/xtensa-lx106-elf/bin/ar")
+if which('openssl') is None and not os.path.isfile('./openssl') and not os.path.isfile('./openssl.exe'):
+    raise Exception("You need to have openssl in PATH, installable from https://www.openssl.org/")# Mozilla's URL for the CSV file with included PEM certs
```

Comment:
Maybe a line break/feed for the comment ?

Reply:
fixed?